addons/2d-color-picker/userscript.js

```diff
@@ -0,0 +1,7 @@
+import paintEditorHandler from "./paint-editor.js";
+
+export default async (api) => {
+  const { addon } = api;
+  await addon.tab.loadScript(addon.self.lib + "/tinycolor-min.js");
+  paintEditorHandler(api);
```

Comment:
Oh, I see. You should not use an arrow function. Instead, use a normal function and pass `arguments[0]`